test/test_skymax.py

```diff
@@ -414,7 +421,9 @@ def test_filtering(self):
             (ht, stilde, psd=self.psd,
              low_frequency_cutoff=self.low_freq_filter, h_norm=1.)
         I_t = I_t * n_t
-        self.assertAlmostEqual(abs(real(I_t.data[idx_max_hom])), max_ds_hom)
+        self.assertAlmostEqual(
+            abs(real(I_t.data[idx_max_hom])), max_ds_hom, places=5
```

Comment:
I had to make it even less string. I also noticed that one of the `chisq` values was calculated but not actually checked. When I added the corresponding assert, I noticed it actually fails (by a lot). So I added it with a FIXME. It may be a genuine bug @spxiwh.